### DISCUSSION TITLE
Node only service resolution

### DISCUSSION THREAD

**Discussion by siredmar** [kubeedge/kubeedge#4128] [Q&A] [answered]
Hi! 
Is such a scenario even possible (with edgemesh)?

1. A DaemonSet that place some pods `A` on several edge nodes
2. One Service that binds to the pods on e.g. `myservice.mynamespace.svc.cluster.local`
3. On the nodes, another pod `B` can access the `A` via its service address `myservice.mynamespace.svc.cluster.local`.  B then connects to A but only on the same node. If A is not running on the node, the access from B will fail.

**Comment by vincentgoat** [comment, marked as answer]
Yes, it is possible with edgemesh.
